docs/tutorials/index.md

```diff
@@ -15,6 +15,7 @@ These tutorials introduce fundamental concepts in deep learning and their realiz
    basic/symbol
    basic/module
    basic/data
+   vision/large_scale_classification
```

Comment:
Why is this under Basics and not under Computer Vision?
In fact we also need to add linear regression to the tutorial index.
Can you rename Computer Vision to "MXNet: Training and Inference"